src/components/WorkshopPage/Workshop.js

```diff
@@ -3,14 +3,15 @@ import ReactPlayer from 'react-player/youtube';
 import Button from '@material-ui/core/Button';
 import Grid from '@material-ui/core/Grid';
 import Typography from '@material-ui/core/Typography';
+import Box from '@material-ui/core/Box';
 import PropTypes from 'prop-types';
 
 function Workshop(props) {
```

Comment:
Can you make this take in a destructured object instead:
`function({ title, youtube, author, description, readme, slides }) {`

Most of the code in this project seem to follow that convention